File: SelectionSort.py
def SelectionSort(a):
    for i in range(len(a)):
        min = i
        for j in range(i+1, len(a)):
            if(a[min]>a[j]):
                min = j
        t = a[i]
        a[i] = a[min]
        a[min]=t

    print("Sorted Array")
    for i in range(len(a)):
        print(a[i],"\t", end = '')

b = [27,10,13,2,104]
SelectionSort(b)

File: SelectionSort.c
#include<stdio.h>
#include<stdlib.h>
void SelectionSort(int A[] , int n)
{

    for(int i = 0;i<n-1;i++)
    {
        int min = i;
        for(int j = i+1;j<n;j++)
        {
            if(A[j]<A[min])
            min = j;
        }
        int t = A[i];
        A[i] = A[min];
        A[min] = t;
    }

    printf("Sorted Array:");
    for(int k = 0;k<=n-1;k++)
    {
        printf("%d \t",A[k]);
    }

}
int main()
{
    int B[] = {6,99,45,23,12,8};
    printf("Array before sorting");
    for(int i=0;i< 6;i++)
        printf("%d  ",B[i]);
    SelectionSort(B,6);
    return 0;

}
